docs/server/source/_static/arch.svg

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
It is just a visual/graphic thing, but can we shift the components slightly to the right-> towards the middle. Currently they are more aligned on the right. Only do it if it is a quick thing.
<img width="611" alt="screen shot 2017-09-26 at 11 09 14 am" src="https://user-images.githubusercontent.com/8788382/30852168-28dc7ec2-a2ab-11e7-81a5-c092ce4f5648.png">

Reply:
Let me see how to do that. Should be do-able.